app/submitter/converter.py

```diff
@@ -120,11 +119,12 @@ def _get_checkbox_answer_data(checkboxes_with_qcode, value):
         option = next((option for option in checkboxes_with_qcode['options'] if option['value'] == user_answer), None)
 
         if option:
-            if option['value'].lower() == 'other':
+            if 'child_answer_id' in option:
+                filtered = answer_store.filter(answer_id=option['child_answer_id'], limit=1)
```

Comment:
This would behave incorrectly if we had a repeating answer but then so would the whole rest of it I believe. Something to note if we ever get something with data version 0.0.1 with repeating